### PR TITLE
raidboss: add ja translations in shb/trial

### PR DESCRIPTION
**Pull request by maiko-tan-coding**
Add / Fix Japanese translation in:
- raidboss_config.js
- shb/titania
- shb/ruby-weapon
- shb/titania-ex
- shb/innocence
- shb/innocence-ex
- shb/hades
- shb/hades-ex

(´。＿。｀) Since I'm not a native Japanese, some translation may not be the best.

### CODE REVIEW

File: ui/raidboss/data/05-shb/trial/hades-ex.js [hunk 1]
```diff
@@ -403,7 +403,7 @@
         en: 'Tank Busters',
         de: 'Tankbuster',
         fr: 'Tankbuster',
-        ja: 'タンクにTB',
+        ja: 'タンクにタンクバスター',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/trial/hades-ex.js:406]: Sorry that I didn't notice that the old text was like this, but where possible can you use the same text from `resources/responses.js` for consistency?

For example, this is maybe `タンクバスター`

File: ui/raidboss/data/05-shb/trial/hades-ex.js [hunk 2]
```diff
@@ -555,15 +555,15 @@
...
+            ja: data.ShortName(matches.target) + 'にタンクバスター',
             cn: '死刑点 ' + data.ShortName(matches.target),
             ko: '"' + data.ShortName(matches.target) + '" 탱버',
           };
         }
         return {
           en: 'Away from ' + data.ShortName(matches.target),
           fr: 'Eloignez-vous de ' + data.ShortName(matches.target),
-          ja: data.ShortName(matches.target) + 'から離れて',
+          ja: data.ShortName(matches.target) + 'から離れ',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/trial/hades-ex.js:566]: The previous text is what is in responses.js as well.

File: ui/raidboss/data/05-shb/trial/hades-ex.js (hunk 1 above)
Comment (maiko-tan-coding) [reply ui/raidboss/data/05-shb/trial/hades-ex.js:406]: OK, I will fix it as the common translation in response.js.

File: ui/raidboss/data/05-shb/trial/hades-ex.js (hunk 2 above)
Comment (maiko-tan-coding) [reply ui/raidboss/data/05-shb/trial/hades-ex.js:566]: Well, I have another opinion about this translation.
I thought `離れ` would be more command-style, just like `Hurry up! Go away!` in English, and `離れて` would be more gently (request-style), like `Please move yourself away from someone`. I prefer the previous one to be the alert text.

**Comment by maiko-tan-coding** [comment]
Merged the newest branch to avoid CI check failed.

File: ui/raidboss/data/05-shb/trial/hades-ex.js (hunk 2 above)
Comment (quisquous) [reply ui/raidboss/data/05-shb/trial/hades-ex.js:567]: That seems reasonable.  Thanks for the explanation!

**Comment by quisquous** [comment]
Thanks!!

**Pull request merged**